recipes/songbird/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set version = "0.8.2" %}
+
+package:
+  name: songbird
+  version: {{ version }}
+
+source:
+  fn: songbird-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/songbird/songbird-{{ version }}.tar.gz
+  sha256: 1e152e5d6114135d0299a362ffd082ca525a6b4f8fc0cc1891ea131892e10a37
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --record=record.txt
```

```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

This is the preferred way of doing the installs these days